### DISCUSSION TITLE
How to copy document with copy its image to outlinewiki 's object storage.

### DISCUSSION THREAD

**Discussion by bestbugwriter** [outline/outline#7955] [Show and tell]
I copied a document with images from the internet and pasted it into my outline. The images are displayed correctly, but the image links are still pointing to the original document. 

I want to ensure that when I copy and paste such a document with images, the images are also copied and stored in my outline's S3 storage. This way, the images won't disappear if the external links become invalid.

[source document](https://www.cnblogs.com/guangdelw/p/18357199)

**Comment by tommoor** [comment]
There's a tracking issue for this here: https://github.com/outline/outline/issues/5713
